assets/tests/state.test.ts

```diff
@@ -0,0 +1,47 @@
+import { Route } from "../src/skate.d"
+import * as State from "../src/state"
```

Comment:
As we get more and more actions, that import list would get pretty long. I did it this way so that the common variables (`initialState` and `reducer`) would be available without needing to update this line every time we add a new action.

Thoughts on the tradeoffs? Is it worth it?

Reply:
Yeah, it looks a little weird to me (like it's a static function call), so I might not use this style but I also don't really care at all.